docs/interpreter/livy.md

```diff
@@ -205,3 +200,7 @@ Edit `conf/spark-blacklist.conf` file in livy server and comment out `#spark.mas
 
 If you choose to work on livy in `apps/spark/java` directory in [https://github.com/cloudera/hue](https://github.com/cloudera/hue),
 copy `spark-user-configurable-options.template` to `spark-user-configurable-options.conf` file in livy server and comment out `#spark.master`. 
+
+## Upgrade
+
+* We remove livy.spark.master in zeppelin-0.7. Because we sugguest user to use livy 0.3 in zeppelin-0.7. And livy 0.3 don't allow to specify livy.spark.master, it enfornce yarn-cluster mode.
```

Comment:
It's just my suggestion; How about putting this sentences under the above Configuartion table with quoted format?